### PR TITLE
st2 [1891][IMP] stock_quantity_history_date: Add date_range_id in stock_account view

### PR DESCRIPTION
**Pull request by TomohiroAbe-Quartile**
[1891](https://www.quartile.co/web#id=1891&action=771&model=project.task&view_type=form&menu_id=505)

Continued from:
#139

### CODE REVIEW

File: stock_quantity_history_date/__manifest__.py [hunk 1]
```diff
@@ -8,6 +8,6 @@
     "author": "Quartile Limited",
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["stock", "date_range"],
+    "depends": ["stock", "stock_account", "date_range"],
```
Comment (yostashiro) [inline stock_quantity_history_date/__manifest__.py]:
```suggestion
    "depends": ["stock_account", "date_range"],
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please rename the file `stock_quantity_view.xml` to `stock_quantity_history.xml`.

ref. https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#complete-structure

File: stock_quantity_history_date/__manifest__.py (hunk 1 above)
Comment (TomohiroAbe-Quartile) [reply stock_quantity_history_date/__manifest__.py]: Thank you, I understand and delete this line.

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Pull request merged**